tests/render/test_template.py

```diff
@@ -0,0 +1,151 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import timezone, datetime, timedelta
+from unittest.mock import MagicMock
+
+import jinja2
+import pytest
+from jinja2 import TemplateNotFound, DictLoader
+from phabricatoremails import PACKAGE_DIRECTORY
+from phabricatoremails.render.events.common import Recipient, Reviewer, ReviewerStatus
+from phabricatoremails.render.events.phabricator import (
+    RevisionCommentPinged,
+    Revision,
+    ReplyContext,
+)
+from phabricatoremails.render.template import (
+    TemplateStore,
+    Template,
+    _jinja_html,
+    _jinja_text,
+)
+
+
+def test_templates_end_with_newline():
+    # The templates need to end with a newline so that the MIME sections are properly
+    # distanced. If they don't have sufficient newlines, then email clients won't
+    # realize that the HTML and text sections are separate.
+    template_dir = PACKAGE_DIRECTORY / "phabricatoremails" / "render" / "templates"
```

Comment:
This path is repeated in other parts of the code (not just in tests), perhaps it should go in its own variable, i.e. `TEMPLATE_DIR`.

Reply:
The way that the template directory is different between tests and production code.
In production, we're using `jinja`'s `PackageLoader`, [which takes a package name and relative path](https://github.com/mozilla-conduit/phabricator-emails/blob/1609266-phab-emails/phabricatoremails/render/template.py#L220):

```
jinja2.PackageLoader("phabricatoremails", "render/templates/html")
```